packages/fiori/src/DynamicSideContent.js

```diff
@@ -0,0 +1,546 @@
+
+// Texts
+import {
+	DSC_SIDE_ARIA_LABEL,
+} from "./generated/i18n/i18n-defaults.js";
+
+/**
+ * @public
+ */
+const metadata = {
+	tag: "ui5-dynamic-side-content",
+	managedSlots: true,
+	properties: /** @lends sap.ui.webcomponents.main.DynamicSideContent.prototype */ {
+
+		/**
+		 * Determines whether to hide the main content.
+		 *
+		 * @type {boolean}
+		 * @public
+		 *
+		 */
+		 hideMainContent: {
+			type: Boolean,
+		},
+
+		/**
+		 * Determines whether to hide the side content.
+		 *
+		 * <b>Note:</b> If both <code>hideSideContent</code> and <code>hideMainContent</code>
+		 * properties are set to <code>false</code>, use the <code>toggleContents</code> method
+		 * for showing the side content on phone.
+		 *
+		 * @type {boolean}
+		 * @public
+		 *
+		 */
+		hideSideContent: {
+			type: Boolean,
+		},
+
+		/**
+		 * Determines whether the side content is on the left or on the right side of the main content.
+		 *
+		 * @type {SideContentPosition}
+		 * @defaultvalue "End"
+		 * @public
+		 *
+		 */
+		 sideContentPosition: {
+			type: SideContentPosition,
+			defaultValue: SideContentPosition.End,
+		},
+
+		/**
+		 * Determines on which breakpoints the side content is visible.
+		 *
+		 * @type {SideContentVisibility}
+		 * @defaultvalue "ShowAboveS"
+		 * @public
+		 *
+		 */
+		sideContentVisibility: {
+			type: SideContentVisibility,
+			defaultValue: SideContentVisibility.ShowAboveS,
+		},
+
+		/**
+		 * Determines on which breakpoints the side content falls down below the main content.
+		 *
+		 * @type {SideContentFallDown}
+		 * @defaultvalue "OnMinimumWidth"
+		 * @public
+		 *
+		 */
+		sideContentFallDown: {
+			type: SideContentFallDown,
+			defaultValue: SideContentFallDown.OnMinimumWidth,
+		},
+
+		/**
+		 * Defines whether the control is in equal split mode. In this mode, the side and
+		 * the main content take 50:50 percent of the container on all screen sizes
+		 * except for phone, where the main and side contents are switching visibility
+		 * using the toggle method.
+		 *
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @public
+		 *
+		 */
+		equalSplit: {
+			type: Boolean,
+		},
+
+		/**
+		 * If set to TRUE, then not the media Query (device screen size) but the size of
+		 * the container, surrounding the control, defines the current range.
+		 *
+		 * @type {boolean}
+		 * @defaultvalue false
+		 * @public
+		 *
+		 */
+		containerQuery: {
+			type: Boolean,
+		},
+
+		/**
+ 		 * @private
+		 */
+		_mcSpan: {
+			type: String,
+			defaultValue: "0",
+			noAttribute: true,
+		},
+
+		/**
+ 		 * @private
+		 */
+		 _scSpan: {
+			type: String,
+			defaultValue: "0",
+			noAttribute: true,
+		},
+
+		/**
+ 		 * @private
+		 */
+		 _toggled: {
+			type: Boolean,
+			noAttribute: true,
+		},
+
+		/**
+ 		 * @private
+		 */
+		 _currentBreakpoint: {
+			type: String,
+			noAttribute: true,
+		},
+
+	},
+	slots: /** @lends sap.ui.webcomponents.main.DynamicSideContent.prototype */ {
```

Comment:
sap.ui.webcomponents.main should be sap.ui.webcomponents.fiori

Reply:
Done